setup-dev.sh

```diff
@@ -37,13 +37,6 @@ if [ -t 1 ]; then
     fi
 fi
 
-if [[ -z "$ZIGPATH" ]]; then
-    # If ZIGPATH is not set, set it to $pwd/.zig
-    # In Docker environment this should by default be set to /opt/zig
-    export ZIGPATH="$(pwd)/.zig"
-fi
-echo "ZIGPATH set to $ZIGPATH"
```

Comment:
We can probably put this in a function and then call it from both install functions, instead of duplicating it.

Reply:
Fixed